### DISCUSSION TITLE
Kvantum manager?

### DISCUSSION THREAD

**Discussion by Aridlvory** [Yisus7u7/termux-desktop-xfce#88] [General]
Can anyone tell me what's this and how to use this. Each time I change the theme all it does is change the theme of the kvantum manager app
![Screenshot_20220218-123940_VNC Viewer.jpg](https://user-images.githubusercontent.com/78092537/154635076-80ed943a-ea91-4cb3-9543-30f1cd12fc56.jpg)

**Comment by JesusChapman** [comment]
> Can anyone tell me what's this and how to use this. Each time I change the theme all it does is change the theme of the kvantum manager app ![Screenshot_20220218-123940_VNC Viewer.jpg](https://user-images.githubusercontent.com/78092537/154635076-80ed943a-ea91-4cb3-9543-30f1cd12fc56.jpg)

You must define the ta of Kvantum manager using the qt5ct application

**Reply by Aridlvory** [reply]
How?
